RELEASING.md

```diff
@@ -254,6 +254,35 @@ openjdk_pipeline.groovy:
 
 Deleting the tag will not fix the problem as it will have been cached on the jenkins worker node used for the trigger jobs - see https://github.com/adoptium/temurin/issues/28#issuecomment-2041364554 for the details, but you'll need to manually adjust build-scripts/utils/./releaseTrigger_jdk*/workspace/tracking
 
+If the git-skara-jdkXXu job in https://ci.adoptium.net/view/git-mirrors/job/git-mirrors/job/adoptium/ gets confused with the dryrun tag, for example
+
+```
+git push origin master --tags
+To github.com:adoptium/jdk23u
+ ! [rejected]                jdk-23.0.2-dryrun-ga -> jdk-23.0.2-dryrun-ga (already exists)
+error: failed to push some refs to 'github.com:adoptium/jdk23u'
+hint: Updates were rejected because the tag already exists in the remote.
+```
+This may be due to an **incorrect** manual tag having been pushed via [step 6](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#dry-run-tests-do-this-at-least-1-week-before-release-in-the-same-calendar-month) rather than letting the mirror job push the tag. To resolve:
```

Comment:
```suggestion

This may be due to an **incorrect** manual tag having been pushed via [step 6](https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#dry-run-tests-do-this-at-least-1-week-before-release-in-the-same-calendar-month) rather than letting the mirror job push the tag. To resolve:
```